ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -103,7 +133,16 @@ tests = testGroup "ChainSyncClient"
 -------------------------------------------------------------------------------}
 
 prop_chainSync :: ChainSyncClientSetup -> Property
-prop_chainSync ChainSyncClientSetup {..} =
+prop_chainSync testSetup@ChainSyncClientSetup {
+    securityParam
+  , clientUpdates
+  , serverUpdates
+  , startTick
+  , invalidBlocks
+  , clientSlowBy
+  } =
+    tabulate "TickArrivalTimeStats" [show (tickArrivalTimeStats traceEvents)] $
```

Comment:
Upon inspection, I was surpsised that ~70% of the cases had `SomeNotEarly_SomeEarly Zero Zero`, which implies the test never involved downloading a header!

My initial guess is that the client and server chain are diverging before they ever reach `startTick`.

I suspect we want to tune the generator to visit that less often. But that can be a separate Issue.

Reply:
Good point, I created #529 for this.

> My initial guess is that the client and server chain are diverging before they ever reach `startTick`.

Yeah, that seems be the case, when always setting `startTick` to `1`, the `Zero Zero` case shrinks to ~9% of all cases.